solitude/settings/products.py

```diff
@@ -0,0 +1,20 @@
+# A definiton of Products for Payments for Firefox Accounts. Marketplace
+# defines these dynamically. However for the moment we will define these from
+# configuration files.
+from decimal import Decimal
+
+products = {
+    'concrete': {
+        'seller': 'mozilla-concrete',
+        'products': [
+            {
+                'name': 'brick',
+                'amount': Decimal('10')
```

Comment:
Yeah i was disappointed to find that we can't create plans in the API, that was the goal of this command, it could now be useless.

But I think a general, "build out the required db values in dev and then check your config in braintree is ok" before we start to do anything might be good. We had a similar script for Bango that proved useful with ops multiple times.